pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -113,6 +113,7 @@ func checkLibvirtInstalled() (bool, error) {
 	return true, nil
 }
 
+// TODO: needs refactor
```

Comment:
With this comment alone, it's not clear what needs to be refactored exactly. Maybe better to leave it out of this commit?

Reply:
part of WIP